### PR TITLE
Enable .com web based login on test builds

### PR DESCRIPTION
**Pull request by crazytonyli**
`.dotComWebLogin` is still in 10% roll out at the moment. Enabling it in test builds would make it easier for others to try the new flow.

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/wordpress-mobile/WordPress-iOS/milestone/303">25.6</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/System/WordPressAppDelegate.swift [hunk 1]
```diff
@@ -540,6 +540,11 @@ extension WordPressAppDelegate {
     /// Updates the remote feature flags using an authenticated remote if a token is provided or an account exists
     /// Otherwise an anonymous remote will be used
     func updateFeatureFlags(authToken: String? = nil, completion: (() -> Void)? = nil) {
+        // Enable certain feature flags on test builds.
+        if BuildConfiguration.current ~= [.a8cPrereleaseTesting, .a8cBranchTest, .localDeveloper] {
```
Comment (kean) [inline WordPress/Classes/System/WordPressAppDelegate.swift:544]: Is this change still needed? I saw the message saying that the is no rush. If that's the case, I'd suggest closing this.

Comment (crazytonyli) [reply WordPress/Classes/System/WordPressAppDelegate.swift:544]: I'm fine with closing this PR. The tricky thing about this particular feature flag is it's on the main login page, where you don't have access to the Debug menu. There is no convenient way to enable the feature flag during testing, without signing into the app first.

Comment (kean) [reply WordPress/Classes/System/WordPressAppDelegate.swift:544]: It's up to you. I'm going to approve.

**Review by kean — APPROVED** [review]
